build/akri-containers.mk

```diff
@@ -10,13 +10,13 @@ include build/akri-python-containers.mk
 #
 # Build the Agent without any Discovery Handlers embedded
 define agent_build_slim
-	PKG_CONFIG_ALLOW_CROSS=1 cross build $(if $(BUILD_RELEASE_FLAG), --release) --target=$(1) --manifest-path agent/Cargo.toml
+	CARGO_INCREMENTAL=$(CARGO_INCREMENTAL) PKG_CONFIG_ALLOW_CROSS=1 cross build $(if $(BUILD_RELEASE_FLAG), --release) --target=$(1) --manifest-path agent/Cargo.toml
 endef
 
 # Build the Agent with features that embed Discovery Handlers and rename the executable in case subsequently
 # building a slim Agent
 define agent_build_with_features
-	PKG_CONFIG_ALLOW_CROSS=1 cross build $(if $(BUILD_RELEASE_FLAG), --release) --target=$(1) --manifest-path agent/Cargo.toml \
+	CARGO_INCREMENTAL=$(CARGO_INCREMENTAL) PKG_CONFIG_ALLOW_CROSS=1 cross build $(if $(BUILD_RELEASE_FLAG), --release) --target=$(1) --manifest-path agent/Cargo.toml \
 	--features "${AGENT_FEATURES}"
 ifeq ($(if $(BUILD_RELEASE_FLAG), --release), --release)
```

Comment:
can probably use the define conditional logic instead of ifeq here ...

```
define agent_build_with_features
	CARGO_INCREMENTAL=$(CARGO_INCREMENTAL) PKG_CONFIG_ALLOW_CROSS=1 cross build $(if $(BUILD_RELEASE_FLAG), --release) --target=$(1) --manifest-path agent/Cargo.toml \
	--features "${AGENT_FEATURES}"
	mv target/$(1)/$(if $(BUILD_RELEASE_FLAG),release,debug)/agent target/$(1)/$(if $(BUILD_RELEASE_FLAG),release,debug)/${FULL_AGENT_EXECUTABLE_NAME}
endef
```

Reply:
Thanks! fixed in 7627f670dad